app/presenters/account_show_presenter.rb

```diff
@@ -38,11 +39,16 @@ def show_gpo_partial?
     decorated_user.pending_profile_requires_verification?
   end
 
+  def show_account_reset_requested_partial?
+    account_reset_requested.present?
```

Comment:
if we move the `.present?` up to all the other cases, this would already be a boolean so we  may not need this method potentially? but it does help readability